example/index.tsx

```diff
@@ -228,6 +231,51 @@ const SingleResourcePdf = () => {
   );
 };
 
+/**
+ * This is an example for passing in a custom getContent callback function to the webreader hook
+ * where the pdf `file` type accepts an `url` string instead of the default BinaryData
+ */
+const LoadPdfWithUrl = () => {
+  const { data: modifiedManifestUrl, isLoading } = useSWR<string>(
+    '/samples/pdf/single-resource-short.json',
+    async (url: string) => {
+      const response = await fetch(url);
+      const manifest = await response.json();
+      console.log(manifest);
```

Comment:
not sure if this is left here intentionally

Reply:
oops - deleting!